packages/gatsby-remark-copy-linked-files/README.md

```diff
@@ -42,3 +73,14 @@ it.
 `my-awesome-pdf.pdf` should be in the same directory as the markdown
 file. When you build your site, the file will be copied to the `public`
 folder and the markdown HTML will be modified to point to it.
+
+### Supported Markdown tags
+
+- img
+- link
+
+### Supported HTML tags
+
+- <img />
+- <video />
+- <a />
```

Comment:
These are not wrapped in code tags `<img/>` and are attempting to render on https://www.gatsbyjs.org/packages/gatsby-remark-copy-linked-files/